### ISSUE TITLE
Add button to open repository

### ISSUE THREAD

**Issue by CodeDoctorDE** [Almighty-Alpaca/JetBrains-Discord-Integration#169]
Add an option to open the git repository like the plugin in vscode:
![image](https://user-images.githubusercontent.com/20452814/123684646-0487f600-d84e-11eb-8860-44bd8da6b139.png)

**Comment by Almighty-Alpaca** [comment]
I really like this idea. Didn't know the button text is customizable.

**Comment by CodeDoctorDE** [comment]
There can be up to 2 buttons. It would be cool if you can customize the second to add there maybe the website of the project?!

**Comment by Almighty-Alpaca** [comment]
Yeah website would be possible too. Another idea was to add an integration with Code With Me (and maybe other 3rd party plugins with equal functionality), but I don't know if we need a join button for that or only implement manually sending invites.

**Comment by CodeDoctorDE** [comment]
I'm a bit confused that the master branch wasn't update sind end 2020

**Comment by Almighty-Alpaca** [comment]
Well.. that's kinda when the last real release was published. There's much more on the development branch. New release is coming soon:tm: after the M1 support has been tested by a few more people.

**Comment by freya022** [comment]
Hey, perhaps you could use the 2nd button to open the file being edited (if not ignored by VCS)

This *could* lead to 404 errors if the file was not commited yet, though

**Comment by CodeDoctorDE** [comment]
I think if the file isn't commited, the second button should be pointed to the folder of the file or could be customized. For example to the production website of the current repository

**Comment by goteusz-maszyk** [comment]
@Almighty-Alpaca Why it's "blocked by discord"?

**Comment by Almighty-Alpaca** [comment]
This is currently marked as "blocked by Discord" because the old discord-rpc library used by the plugin doesn't support custom buttons. There was an attempt to move to the newer GameSDK (mainly in hope of fewer connection issues), but that one doesn't support custom buttons as well ([last time I checked](https://github.com/discord/gamesdk-and-dispatch/issues/123)).

**Comment by 4mrcn4** [comment]
Can this get some love? Or do we really have to use deprecated API to implement this?

**Comment by PrincessAkira** [comment]
Is this still beeing worked on e.g have you found a library that supports buttons?
